kubernetes-aws-yaml/Pulumi.yaml

```
@@ -0,0 +1,52 @@
+name: k8s-aws-yaml
+runtime: yaml
+description: A Pulumi YAML program to stand up a Kubernetes cluster on AWS
+template:
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
```

Replacing these with expressions will swap in these user-defined values as project creation-time:

```suggestion
name: ${PROJECT}
description: ${DESCRIPTION}
runtime: yaml

template:
  description: A Pulumi YAML program to stand up a Kubernetes cluster on AWS
  config:
    aws:region:
      description: The AWS region to deploy into
      default: us-west-2
```

---

Changes incorporated via e084496